### DISCUSSION TITLE
可莉特调变强了

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#780] [General]
角色配装界面门栓拍图都能识别……

**Comment by Arondight** [comment]
白嫖人家的东西，有必要做个宣传，晚上写一下

**Comment by mark9804** [comment]
确实变强了
<img width="628" alt="image" src="https://user-images.githubusercontent.com/9006264/161970204-08eeba21-0212-4462-8244-8be68ebc78d6.png">
